PPO/PPO.py

```diff
@@ -160,12 +171,15 @@ def learn(self, num_learn, last_value, next_done):
       # calculate gradient
       self.optimiser.zero_grad()
       loss.backward()
-      nn.utils.clip_grad_norm_(self.model_old.parameters(), 0.5)
+      nn.utils.clip_grad_norm_(self.model.parameters(), 0.5)
```

Comment:
@DarylRodrigo I need you to look at this, I hope I didn't misunderstand something.